plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/CountingAccessHiveMetastore.java

```diff
@@ -341,6 +344,7 @@ public void updatePartitionStatistics(Table table, Map<String, Function<Partitio
     @Override
     public List<String> getAllTables(String databaseName)
     {
-        throw new UnsupportedOperationException();
+        methodInvocations.add(Methods.GET_ALL_TABLES_WITH_PARAMETER);
+        return delegate.getAllTables(databaseName);
```

Comment:
why `_WITH_PARAMETER`? shouldn't this be `GET_ALL_TABLES`?

Reply:
In PR https://github.com/trinodb/trino/pull/17127 I will add `getAllTables` without any parameter

Reply:
Renamed